src/main/java/frc/robot/subsystems/drivetrain/Drive.java

```diff
@@ -26,6 +26,7 @@ public class Drive extends SubsystemBase {
     private final Module rearRight;
 
     private final SwerveDriveKinematics kinematics;
+    public Rotation2d Angle;
```

Comment:
angle (should be lowercased)
And also you don't use it

Reply:
wiil be deleted.